xml/System.Security.Cryptography/CryptoStream.xml

```diff
@@ -591,6 +591,7 @@ Copying begins at the current position in the current stream, and does not reset
         <exception cref="T:System.ObjectDisposedException">Either the current stream or <paramref name="destination" /> were closed before the <see cref="M:System.IO.Stream.CopyTo(System.IO.Stream)" /> method was called.</exception>
         <exception cref="T:System.IO.IOException">An I/O error occurred.</exception>
         <exception cref="T:System.Security.Cryptography.CryptographicException">An error occurred during a cryptographic operation.</exception>
+        <inheritdoc />
```

Comment:
This method has full documentation... what does inheritdoc mean in context?

Reply:
This is what we should've ported from the beginning, instead of the individual texts. I'm adding these since the tool now detects them. We can later decide if we want to clear the summary, returns, params, typeparams of these so that we use the inherited documentation.